pkg/cmd/pulumi/deployment_settings_utils_test.go

```diff
@@ -56,22 +57,15 @@ func TestRepoLookup(t *testing.T) {
 
 	t.Run("should handle directories that are a git repo", func(t *testing.T) {
 		t.Parallel()
-		ctx := context.Background()
 
-		repo := auto.GitRepo{
-			URL:         "https://github.com/pulumi/test-repo.git",
-			ProjectPath: "goproj",
-			Shallow:     true,
-			Branch:      "master",
-		}
-		ws, err := auto.NewLocalWorkspace(ctx, auto.Repo(repo))
-		require.NoError(t, err)
+		repoDir := setUpGitWorkspace(context.Background(), t)
```

Comment:
tests are not performing changes in the filesystem so we can reuse the same workspace for all of them